doc/examples/features_detection/plot_orb.py

```diff
@@ -49,8 +49,17 @@
 
 plot_matches(ax[0], img1, img2, keypoints1, keypoints2, matches12)
 ax[0].axis('off')
+ax[0].set_title("Original Image VS Transformed Image")
+print('Transformed Image using Rotation = 180 degrees')
 
 plot_matches(ax[1], img1, img3, keypoints1, keypoints3, matches13)
 ax[1].axis('off')
+ax[1].set_title("Original Image VS Transformed Image")
+print('Transformed Image using AffineTransform parameters:')
```

Comment:
In my opinion, this logging information does not add value to the example... @scikit-image/core Can a second reviewer please contribute their thoughts?